recipes/spirv-headers/all/conandata.yml

```diff
@@ -1,49 +1,16 @@
 sources:
+  "1.3.268.0":
+    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.268.0.tar.gz"
+    sha256: "1022379e5b920ae21ccfb5cb41e07b1c59352a18c3d3fdcbf38d6ae7733384d4"
+  "1.3.261.1":
+    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.261.1.tar.gz"
+    sha256: "32b4c6ae6a2fa9b56c2c17233c8056da47e331f76e117729925825ea3e77a739"
   "1.3.243.0":
     url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.243.0.tar.gz"
     sha256: "16927b1868e7891377d059cd549484e4158912439cf77451ae7e01e2a3bcd28b"
   "1.3.239.0":
     url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.239.0.tar.gz"
     sha256: "fdaf6670e311cd1c08ae90bf813e89dd31630205bc60030ffd25fb0af39b51fe"
-  "1.3.236.0":
-    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.236.0.tar.gz"
-    sha256: "4d74c685fdd74469eba7c224dd671a0cb27df45fc9aa43cdd90e53bd4f2b2b78"
-  "1.3.231.1":
-    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.231.1.tar.gz"
-    sha256: "fc340700b005e9a2adc98475b5afbbabd1bc931f789a2afd02d54ebc22522af3"
-  "1.3.224.0":
-    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.224.0.tar.gz"
-    sha256: "2fb1039ec6cec8943400e9b4d01d8bfe8c62a0bd1fafb7c39c56469aa247b838"
-  "1.3.216.0":
-    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.216.0.tar.gz"
-    sha256: "46c49a0e49ea120138102b1dcb3778e5a4f2267c45b9e937810a4cf4fb889e3d"
-  "1.3.211.0":
-    url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.211.0.tar.gz"
-    sha256: "30a78e61bd812c75e09fdc7a319af206b1044536326bc3e85fea818376a12568"
```

Comment:
Shouldn't we keep these? They are still used in [spirv-tools](https://github.com/conan-io/conan-center-index/blob/master/recipes/spirv-tools/config.yml)

Reply:
I'll remove these versions in spirv-tools once this PR is merged.